src/frontend/web_application/src/scenes/Discussion/components/MessagePi/index.jsx

```diff
@@ -88,21 +88,23 @@ class MessagePi extends PureComponent {
     return (
       <div className="m-message-pi">
         {illustrate ? this.renderIllustration() : null}
-        <div
-          className={classnames(['m-message-pi__progress', `m-message-pi__progress--${getPiClass(piAggregate)}`])}
-          role="progressbar"
-          aria-valuenow={piAggregate}
-          aria-valuemax="100"
-          tabIndex="0"
-        >
+        <div className="m-message-pi__meter">
           <div
-            className={classnames('m-message-pi__progress-meter', `m-message-pi__progress-meter--${getPiClass(piAggregate)}`)}
-            style={{ width: `${piAggregate}%` }}
-          />
-        </div>
-        <div className="m-message-pi__numeric">
-          <span className="legend">Privacy index&thinsp;:</span>
-          <span className="value">{Math.round(piAggregate)}</span>
+            className={classnames(['m-message-pi__progress', `m-message-pi__progress--${getPiClass(piAggregate)}`])}
+            role="progressbar"
+            aria-valuenow={piAggregate}
+            aria-valuemax="100"
+            tabIndex="0"
+          >
+            <div
+              className={classnames('m-message-pi__progress-meter', `m-message-pi__progress-meter--${getPiClass(piAggregate)}`)}
+              style={{ width: `${piAggregate}%` }}
+            />
+          </div>
+          <div className="m-message-pi__numeric">
+            <span className="m-message-pi__numeric--legend">Privacy index&thinsp;:</span>
```

Comment:
`legend` & `value` are not modifiers like describe in BEM notation, it should be named:

```
m-message-pi__numeric-legend
m-message-pi__numeric-value
```